operator/internal/manifests/config.go

```diff
@@ -261,3 +240,15 @@ func retentionConfig(ls *lokiv1.LokiStackSpec) config.RetentionOptions {
 		DeleteWorkerCount: deleteWorkerCountMap[ls.Size],
 	}
 }
+
+// ConfigureOptionsForMode applies configuration depending on the mode type.
+func ConfigureOptionsForMode(cfg *config.Options, mode lokiv1.ModeType) error {
+	switch mode {
+	case lokiv1.Static, lokiv1.Dynamic:
+		return nil // nothing to configure
+	case lokiv1.OpenshiftLogging, lokiv1.OpenshiftNetwork:
+		return openshift.ConfigureOptions(cfg)
+	}
+
+	return nil
+}
```

Comment:
Please move this to the `gateway_tenants.go` file where all the other configure function live.

Reply:
why gateway_tenants? it has nothing to do with the gateway..

Reply:
Actually the filename is wrong nowadays it should be more `tenants.go`

Reply:
ok I see, I will rename it then to avoid  confusion

Reply:
Let's rename in a separate PR, because this is a crucial file that will break many PRs right now.